crops/crop.go

```diff
@@ -115,3 +124,36 @@ func (c Crop) ComputeConsequences(event interface{}) consequences.Results {
 	r := consequences.Results{IsTable: false, Result: ret}
 	return r
 }
+func (c Crop) computeImpactedCase(e hazards.ArrivalandDurationEvent) float64 {
+	// Determine damage percent based on damage dur curve and event dur
+	perdmg := c.lossFunction.ComputeDamagePercent(e)
+	fmt.Println("damage percent is ", perdmg)
+	croploss := (perdmg / 100) * c.GetValuePerOutputUnit()
+	fmt.Println("Crop loss is : ", croploss)
+	// value added to field before loss by production
+	fmt.Println("total pruduction costs are:", c.productionFunction.productionCostLessHarvest)
+	fmt.Println("fixed costs are: ", c.productionFunction.GetfixedCosts())
+	loss := croploss + c.productionFunction.productionCostLessHarvest
+	fmt.Println("total loss is:", loss)
+	if loss > c.valuePerOutputUnit {
+		// Throw some error
+	}
+	return 10
+}
+func (c Crop) computeDelayedCase() float64 {
+	// delayed loss is equivalent to total marketable value less harvest cost, times the percent loss due to late planting
+	// Not using interpolated % loss for late plant
+	valuelessharvest := (c.GetValuePerOutputUnit()) - c.productionFunction.harvestCost
+	croploss := (valuelessharvest) * c.productionFunction.lossFromLatePlanting
+	fmt.Println("total loss is : ", croploss)
+	return croploss
+}
+
+func (c Crop) computeNotPlantedCase() float64 {
+	// Assume Loss is only fixed costs
+	return c.productionFunction.GetfixedCosts()
```

Comment:
refactor "GetTotalFixedCosts"? fixed costs is an array. so you would want to describe total across the planting season or the year?

Reply:
commit 7d3a87da68e6c112a665eda95d671176ab40f6b3 changes names of methods and variables to reflect total fixed costs. 

field is still fixed cost to be consistent with productionFunction.harvestCost and productionFunction,productionCostLessHarvest which are totals not the arrays